### PR TITLE
schemas: add noAdditionalProperties everywhere

### LINKED ISSUE #26: schemas: add noAdditionalProperties everywhere
Continues https://github.com/inspirehep/inspire-next/pull/1504.

### PR DESCRIPTION
**Pull request by kaplun**
* BETTER Adds noAdditionalProperties everywhere.
  (closes #26)

Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

File: inspire_schemas/records/authors.json [hunk 1]
```diff
@@ -264,11 +271,12 @@
...
-                "date_verified": {
-                    "format": "date",
-                    "type": "string"
-                },
+                "noAdditionalProperties": true,
                 "properties": {
+                    "date_verified": {
+                        "format": "date",
+                        "type": "string"
+                    },
```
Comment (kaplun) [inline inspire_schemas/records/authors.json]: This I realized was a misplaced property.

**Comment by jacquerie** [comment]
This needs a big fat incompatible tag...

**Comment by kaplun** [comment]
Does it? I think it's compatible. Rules that have generated non-defined properties where erring on the wrong side anyway.

But sure, I can add the incompatible tag.

**Comment by jacquerie** [comment]
Mostly for dividing development in two: first we generate what's in the
schema at version N, then in version N+1 we avoid what's not in the schema.

Otherwise you have to do both at the same time...

**Pull request merged**